ydb/docs/en/core/changelog-server.md

```diff
@@ -1,5 +1,85 @@
+  * `HASH` — set a password in the encrypted form.
+  * `LOGIN` and `NOLOGIN` — unlock and block a user.
+* Enhanced account security:
+  * [Added](https://github.com/ydb-platform/ydb/pull/11963) user password complexity verification.
+  * [Implemented](https://github.com/ydb-platform/ydb/pull/12578) automatic user lockout after a specified number of failed attempts to enter the correct password.
+  * [Added](https://github.com/ydb-platform/ydb/pull/12983) the ability for users to change their own passwords.
+* [Implemented](https://github.com/ydb-platform/ydb/issues/9748) the ability to toggle functional flags at runtime. Changes to the flags, for which the `(RequireRestart) = true` parameter  is not specified in the [proto file](https://github.com/ydb-platform/ydb/blob/main/ydb/core/protos/feature_flags.proto#L60), will be applied without a restart.
+* [Changed](https://github.com/ydb-platform/ydb/pull/11329) lock behavior when the number of locks on shards exceeds the limit. Now, once the limit is exceeded, the oldest locks (rather than the newest) are converted into full-shard locks.
+* [Implemented](https://github.com/ydb-platform/ydb/pull/12567) a mechanism to preserve optimistic locks in memory when datashards are gracefully restarted, which should reduce the number of ABORTED errors due to lock loss during table balancing between nodes.
+* [Implemented](https://github.com/ydb-platform/ydb/pull/12689) a mechanism to abort volatile transactions with the ABORTED status when datashards are gracefully restarted.
+* [Added](https://github.com/ydb-platform/ydb/pull/6342) the ability to remove NOT NULL constraints on a column in a table using the `ALTER TABLE _ ALTER COLUMN _ DROP NOT NULL` query.
+* [Added](https://github.com/ydb-platform/ydb/pull/9168) the 100.000 limit on the number of simultaneous requests to create sessions in the coordination service.
+* [Increased](https://github.com/ydb-platform/ydb/pull/14219) the maximum [number of columns in the primary key](./concepts/limits-ydb.md?#schema-object) from 20 to 30.
+* **_(Experimental)_** [Added](https://github.com/ydb-platform/ydb/pull/14075) strict access control checks, which are enabled by setting the `enable_strict_acl_check` flag and the following ones:
+  * `enable_strict_user_management` enables strict checks for local users.
+  * `enable_database_admin` enables database administrator functions.
+  * `enable_data_erasure` [enables](https://github.com/ydb-platform/ydb/pull/14460) a procedure that repeatedly overwrites deleted data to minimize the risk of recovering the deleted information when directly accessing block devices via operating system functions.
+
+#### Backward incompatible changes
+
+* If you are using [temporal over YDB](https://github.com/yandex/temporal-over-ydb), update it to version [v1.23.0-ydb-compat](https://github.com/yandex/temporal-over-ydb/releases/tag/v1.23.0-ydb-compat) before updating YDB to the current version to avoid errors in query execution.
```

Comment:
❗As far as I understand, this project just queries YDB in some way via the Go SDK. If it's no longer compatible, then some behavior of YDB visible via the public API has changed. This behavior change needs to be documented, not just the need to update an external project. Otherwise, any other client doing similar queries would get broken too, without a clear warning and instructions to avoid or fix.